File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/response/InventarioUbicarProductoResponse.java
package com.besoft.siserp.lgtc.dto.response;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class InventarioUbicarProductoResponse implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idProducto;
	private Long idEstante;
	private String nomEstante;
	private Long idAlmacen;
	private String nomAlmacen;
	private Double cantidad;
	private Double precio;
	private Integer flagActivo;

}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/response/inventario-ubicar-producto.response.ts
import { AlmacenListarResponse } from "./almacen-listar.response";

export class InventarioUbicarProductoResponse {
    idProducto: number;
    idEstante: number;
    nomEstante: string;
    idAlmacen: number;
    nomAlmacen: string;
    cantidad: number;
    precio: number;
    flagActivo: number;
}